src/producer.ts

```diff
@@ -1,5 +1,6 @@
 import { SQS } from 'aws-sdk';
 import { toEntry } from './types';
+import { SendMessageBatchResultEntryList } from 'aws-sdk/clients/sqs';
```

Comment:
I think here the lint would complain as it needs to be alphabetical order. Thanks